api/serializers.py

```diff
@@ -1833,6 +1841,49 @@ def get_user_countries_regions(user):
         qs = UserCountry.objects.filter(user=user).distinct("country")
         return UserCountrySerializer(qs, many=True).data
 
+    @staticmethod
+    def get_local_unit_global_validators(user) -> List[int]:
+        data = list(
+            Permission.objects.filter(codename__startswith="local_unit_global_validator", group__user=user).values_list(
+                "codename", flat=True
+            )
+        )
+        global_validators = []
+        for code in data:
+            type_id = code.split("_")[-1]
+            global_validators.append((int(type_id)))
+        return global_validators
```

Comment:
Use Python Set Comprehensions

```suggestion
    def get_local_unit_global_validators(user) -> list[int]:
      permission_codenames = Permission.objects.filter(
          codename__startswith="local_unit_global_validator",
          group__user=user
      ).values_list("codename", flat=True)

      global_validators = {int(code.split("_")[-1]) for code in permission_codenames}
      return list(global_validators)
```